packages/wonder-blocks-core/src/hooks/__tests__/use-on-mount-effect.test.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import {renderHook} from "@testing-library/react-hooks";
+
+import {useOnMountEffect} from "../use-on-mount-effect.js";
+
+describe("#useOnMountEffect", () => {
+    it("shoul call the callback once", () => {
```

Comment:
**nit:** 
```suggestion
    it("should call the callback once", () => {
```

Reply:
**change:** Arrange/Act/Assert comments are missing from these test cases.

See best practices: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/98402353/JavaScript+Testing+Best+Practices

Reply:
Sorry about that.  I'll add them in.

Reply:
**aside:** this is totally the kind of thing we could have an action for that checks for the comments and if there missing as the link in a comment.